v2/tools/generator/internal/codegen/pipeline/create_arm_types.go

```diff
@@ -8,6 +8,8 @@ package pipeline
 import (
 	"context"
 
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/config"
```

Comment:
minor: ordering, should be at the bottom with the other v2/tools imports.

Reply:
Fixed.